tests/test_ir/test_key_value_pair_log_event.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+from typing import Any, Dict
+
+from test_ir.test_utils import JsonLinesFileReader, TestCLPBase
+
+from clp_ffi_py.ir import KeyValuePairLogEvent
+
+
+class TestCaseKeyValuePairLogEvent(TestCLPBase):
+    """
+    Class for testing `clp_ffi_py.ir.KeyValuePairLogEvent`.
+    """
+
+    test_data_dir: Path = Path("test_data")
+    jsonl_dir: Path = Path("jsonl")
+
+    def test_basic(self) -> None:
+        """
+        Tests the conversion between a Python dictionary and a `KeyValuePairLogEvent` instance,
+        ensuring accurate serialization and deserialization in both directions.
+        """
+        current_dir: Path = Path(__file__).resolve().parent
+        test_src_dir: Path = (
```

Comment:
This should really be called `test_data_dir`, right?

Reply:
yeah make sense